### ISSUE TITLE
Your application folder path does not appear to be set correctly. Please open the following file and correct this: app_dev.php

### ISSUE THREAD

**Issue by pedroresende** [evispa/NercuryCodeIgniterBundle#9]
Tried to follow the readme.md file, I've configured everything but I'm getting the following when I try to use

```
$response = $this->get("ci")->getResponse($request);
```

in a controller, any ideas ?

**Comment by Nercury** [comment]
Looks like you are still running it over old CI, not symfony, because in this bundle CI bootstrap lines that would print this message are commented out (see https://github.com/evispa/NercuryCodeIgniterBundle/blob/4afa1b7c18374744345ad8b763df2d62b3618623/CiHelperService.php#L176).

**Comment by pedroresende** [comment]
@Nercury thanks, but in that case how should I do it ? I've followed the README instructions. If I try to access one of the "old" routes it doesn't work, event though I didn't added the `detect_controllers: false` option.
What I have done, is to add a Symfony Controller with an "old" route in order to be catched by Symfony. Then, in that controller I've tried to use

``` 
$response = $this->get("ci")->getResponse($request);
```

Any idea how to do it correctly ? 

Thank in advance for your help :)
